netci.groovy

```diff
@@ -29,9 +27,7 @@ def osShortName = ['Windows 7' : 'win7',
                    'Windows_NT' : 'windows_nt',
                    'Ubuntu14.04' : 'ubuntu14.04',
                    'Ubuntu16.04' : 'ubuntu16.04',
-                   'Ubuntu16.10' : 'ubuntu16.10',
                    'Debian8.4' : 'debian8.4',
-                   'Fedora24' : 'fedora24',
                    'OSX10.12' : 'osx',
                    'CentOS7.1' : 'centos7.1',
                    'RHEL7.2' : 'rhel7.2',
```

Comment:
RHEL 7.2 was EOL last year, any reason you left it out?

Reply:
I was on the fence on whether we should remove RHEL72. How does azure feel about spinning up RHEL72 VM's?

Reply:
Azure has no feelings about it, but the publisher can pull the image or azure package source mirrors at any time;  if they do so, having the VMs in Azure is much harder (not impossible, but the point of the work we're doing together is to get rid of the EOL OSes)

Reply:
I see. May as well remove it then.